src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    @Autowired
+    private RoleService roleService;
+    @Autowired
+    private UserService userService;
+
+    @PostConstruct
+    public void inject() {
```

Comment:
Method name 'inject' can be more descriptive. Consider renaming it to something like 'initializeRolesAndUsers'.